pkg/localize/locales/en/cmd/cluster_bind.en.toml

```
@@ -29,6 +29,9 @@ $ rhoas cluster bind --namespace=ns --app-name=myapp
 [cluster.bind.flag.appName]
 one = '''Name of the kubernetes deployment to bind'''
 
+[cluster.bind.flag.bindName]
+one = '''Name of the Service binding object to create when using operator'''
```

```suggestion
one = 'Name of the Service binding object to create when using operator'
```

The three quotes are only necessary for multi-line strings.